src/gt4py/next/ffront/lowering_utils.py

```diff
@@ -79,7 +79,7 @@ def to_iterator_of_tuples(expr: itir.Expr | str, arg_type: ts.TypeSpec) -> itir.
         for type_ in type_constituents
     )
 
-    def fun(_, path):
+    def fun(_: Any, path: tuple[int, ...]) -> itir.FunCall:
```

Comment:
Just a question: are we sure `path` is a tuple of ints? Shouldn't also be allowed to contain `str`?

Reply:
my understanding is that it is only `int`s